docs/creating-a-runtime-check.md

```diff
@@ -0,0 +1,181 @@
+# Creating a Runtime Check
+
+Runtime checks are used to perform tests by executing the plugins code.
+
+The Plugin Checker uses a number of interfaces to build out runtime checks.
+
+One interface is the `Runtime_Check` interface. This interface contains no methods, but is used to help identify a runtime check from static checks.
```

Comment:
Related to my feedback on the static check docs, maybe clarify that this is the only _requirement_ for a runtime check to implement, as well as accompany with a very quick code example like `Custom_Check implements Runtime_Check {...`